### PR TITLE
[PIN-1677] Adding limitations to strings fields inside yml

### PR DESCRIPTION
**Pull request by bloodsky**

### CODE REVIEW

**Review by TonioGela — APPROVED** [review]

File: src/main/resources/interface-specification.yml [hunk 1]
```diff
@@ -501,8 +502,10 @@ components:
       properties:
         origin:
           type: string
+          maxLength: 64
         code:
           type: string
+          maxLength: 30
```
Comment (galales) [inline src/main/resources/interface-specification.yml:511]:
```suggestion
          minLength: 3
          maxLength: 30
```

File: src/main/resources/interface-specification.yml [hunk 2]
```diff
@@ -511,6 +514,7 @@ components:
       properties:
         code:
           type: string
+          maxLength: 30
```
Comment (galales) [inline src/main/resources/interface-specification.yml:521]:
```suggestion
          minLength: 3
          maxLength: 30
```

File: src/main/resources/interface-specification.yml [hunk 3]
```diff
@@ -780,8 +790,11 @@ components:
           $ref: '#/components/schemas/MailKind'
         address:
           type: string
+          maxLength: 30
```
Comment (galales) [inline src/main/resources/interface-specification.yml]:
```suggestion
          minLength: 5
          maxLength: 320
```

File: src/main/resources/interface-specification.yml [hunk 4]
```diff
@@ -793,11 +806,13 @@ components:
           $ref: '#/components/schemas/MailKind'
         address:
```
Comment (galales) [inline src/main/resources/interface-specification.yml:812]: this should be an output-only model

**Review by galales — APPROVED** [review]

**Pull request merged**